src/fiskaltrust.ifPOS/v0/Models/SignaturItem.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v0
+{
+    /// <summary>
+    /// The signature entry is only used for the receipt response.
+    /// </summary>
+    [DataContract]
+    public partial class SignaturItem
+    {
+        public enum Types : long
+        {
+            /// <summary>
```

Comment:
There are many empty XML tags in these models. Should we get rid of them, or keep this as-it-is since it's anyhow v0?